modules/ppcp-settings/resources/js/Components/Screens/Onboarding/Components/AdvancedOptionsForm.js

```diff
@@ -29,38 +29,13 @@ const AdvancedOptionsForm = ( { setCompleted } ) => {
 	const refClientId = useRef( null );
 	const refClientSecret = useRef( null );
 
-	const handleFormValidation = () => {
-		const fields = [
-			{
-				ref: refClientId,
-				value: clientId,
-				errorMessage: __(
-					'Please enter your Client ID',
-					'woocommerce-paypal-payments'
-				),
-			},
-			{
-				ref: refClientSecret,
-				value: clientSecret,
-				errorMessage: __(
-					'Please enter your Secret Key',
-					'woocommerce-paypal-payments'
-				),
-			},
-		];
-
-		for ( const { ref, value, errorMessage } of fields ) {
-			if ( value ) {
-				continue;
-			}
-
-			ref?.current?.focus();
-			createErrorNotice( errorMessage );
-			return false;
-		}
+	const isValidClientId = useMemo( () => {
+		return clientId ? /^A[\w-]{79}$/.test( clientId ) : true;
```

Comment:
It seems to me that this returns `true` if `clientId` does not exist... if so, why?

Reply:
@Dinamiko It was to prevent showing validation errors when clientId is empty, but we are also checking that on the field itself at the moment of assigning the class, so this is redundant. I have changed it on the latest commit 👍🏻